mb_modelbase/server/modelbase.py

```diff
@@ -147,7 +151,7 @@ class ModelBase:
             .float_format : The float format used to encode floats in a result. Defaults to '%.5f'
     """
 
-    def __init__(self, name, model_dir='data_models', load_all=True, watchdog=True):
+    def __init__(self, name, model_dir='data_models', load_all=True, cache=None, watchdog=True):
         """ Creates a new instance and loads models from some directory. """
```

Comment:
Please add documentation. I know it's not all your code, so thanks :)

Reply:
Done! No problem :-)